### ISSUE TITLE
wrong indentation when using a multi line `modloadmsg` with Lua format

### ISSUE THREAD

**Issue by pescobar** [easybuilders/easybuild-framework#2405]
I have noticed that if define this in an easyconfig:
```
modloadmsg = "To execute Pilon run: java -Xmx16G -jar $EBROOTPILON/pilon.jar \n"
modloadmsg += "Adjust the memory value according to the size of your input files \n"
```

The indentation is wrong when loading the module:
```
$ ml Pilon
To execute Pilon run: java -Xmx16G -jar $EBROOTPILON/pilon.jar 
    Adjust the memory value according to the size of your input files 
    [user@host]$
```

This is the generated Lua modulefile:

```
if mode() == "load" then
    io.stderr:write([==[To execute Pilon run: java -Xmx16G -jar $EBROOTPILON/pilon.jar
    Adjust the memory value according to the size of your input files
    ]==])
end
```

If I modify the modulefile like this then it works ok:
```
if mode() == "load" then
    io.stderr:write([==[
    To execute Pilon run: java -Xmx16G -jar $EBROOTPILON/pilon.jar
    Adjust the memory value according to the size of your input files
]==])
end
```
It would be cool if easybuild would generate the modulefiles with the proper indentation.

By now I am using a workaround suggested by Zao in the chat using `\r`

```
modloadmsg = "To execute Pilon run: java -Xmx8G -jar $EBROOTPILON/pilon.jar\n"
modloadmsg += "\rAdjust the memory value according to the size of your input files\n\r"
```

**Comment by pescobar** [comment]
I am now sure what changed but with latest easybuild-3.8.1 this format seems to work fine to workaround this issue:

```
modloadmsg = """
To execute GATK run 'gatk'
To enable bash completion run 'source $EBROOTGATK/gatk-completion.sh'
"""
```

I am using Lmod

```
Modules based on Lua: Version 7.8.11  2018-11-28 14:08 -06:00
    by Robert McLay mclay@tacc.utexas.edu
```

**Comment by boegel** [comment]
@pescobar The indent issue was fixed in #2613

**Comment by pescobar** [comment]
@boegel I owe you one more (belgian) beer   ;)
